config/environments/test.rb

```diff
@@ -13,7 +13,7 @@
   config.eager_load = false
 
   # Configure static asset server for tests with Cache-Control for performance.
-  config.serve_static_assets  = true
+  config.serve_static_files  = true
```

Comment:
The deprecation warning is shows that serve_static_assets is an old name, and the new name is serve_static_files. Rails makes steady improvements to names and methods, so this is a pretty common warning.